web/themes/custom/digital_gov/templates/node/node--landing-page--job-board.html.twig

```diff
@@ -82,7 +81,59 @@
 
   <section class="stream">
     <div class="grid-container-desktop">
-      [placeholder list job cards]
+
+  {% for item in job_board %}
```

Comment:
Please look more closely how the source site handles no active / empty job boards
https://github.com/GSA/digitalgov.gov/blob/a894f758f60e574c56e8ac1701c307d2b1f47949/themes/digital.gov/layouts/job-board/list.html#L45
Right now, this is what happens for ours:
![image](https://github.com/user-attachments/assets/1828d628-80e8-4663-a27d-8c303a16478d)